Extensions/Terraform/Src/Tasks/TerraformTask/TerraformTaskV2/src/base-terraform-command-handler.ts

```diff
@@ -204,6 +204,7 @@ export abstract class BaseTerraformCommandHandler {
 
     public async plan(): Promise<number> {
         await this.onlyPlan();
+        tasks.setVariable('changesPresent', exitCode === 2);
```

Comment:
does exitCode need to be defined here?
Something like
`let exitCode = await this.onlyPlan();`